lua/lspinstall/servers/puppet.lua

```diff
@@ -0,0 +1,13 @@
+local config = require"lspinstall/util".extract_config("puppet")
+config.default_config.cmd[1] = "./puppet-editor-services/puppet-languageserver"
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
+    name=puppet-editor-services
```

Comment:
just write `puppet-editor-services` everywhere instead of `$name`

Reply:
Also changed now.